Services/WordCount/WordCount.Service/Controllers/DefaultController.cs

```diff
@@ -50,9 +54,47 @@ public DefaultController(IReliableStateManager stateManager)
                 await queue.EnqueueAsync(tx, word);
 
                 await tx.CommitAsync();
+
+                await UpdateStats();
             }
 
             return this.Ok();
-        }
+        }
+
+        private async Task<long> UpdateStats(int count = 1)

```

Comment:
I like your solution that you are trying to update the stats from inside the AddWord method. But I cannot understand why the same code block is leaved unchanged in DefaultController.cs? Have a missed something? If I comment out the code block in DefaultController it works as expected but the CPU are running at 100% instead of approx 40% in the original code.